src/assets/Doneth.sol.txt

```diff
@@ -227,6 +255,57 @@ contract Doneth {
 }
 
 /**
+ * @title ERC20Basic
+ * @dev Simpler version of ERC20 interface
+ * @dev see https://github.com/ethereum/EIPs/issues/179
+ */
+contract ERC20Basic {
+  uint256 public totalSupply;
+  function balanceOf(address who) public constant returns (uint256);
+  function transfer(address to, uint256 value) public returns (bool);
+  event Transfer(address indexed from, address indexed to, uint256 value);
+}
+
+
+/**
+ * @title ERC20 interface
+ * @dev see https://github.com/ethereum/EIPs/issues/20
+ */
+contract ERC20 is ERC20Basic {
+  function allowance(address owner, address spender) public constant returns (uint256);
+  function transferFrom(address from, address to, uint256 value) public returns (bool);
+  function approve(address spender, uint256 value) public returns (bool);
+  event Approval(address indexed owner, address indexed spender, uint256 value);
+}
+
+
+/*
+ * Ownable
+ *
+ * Base contract with an owner.
+ * Provides onlyOwner modifier, which prevents function from running if it is called by anyone other than the owner.
+ */
+
+contract Ownable {
```

Comment:
Can you explain your vision for including this? It seems like it is a replacement for the "founder" concept in our current code. If so, shouldn't we also clean up the founder logic and replace it with owner logic?

Reply:
yup that's what i was thinking too, sorry for so many loose ends in this branch, meant it as a starting point. I figured since ownable does the exact same thing as founder we might as well adhere to the community standard and replace founder with it.